cli/azd/pkg/executil/executil.go

```diff
@@ -197,10 +198,31 @@ type RunArgs struct {
 	EnrichError bool
 }
 
+type RedactData struct {
+	matchString   *regexp.Regexp
+	replaceString string
+}
+
 func (rr RunResult) String() string {
 	return fmt.Sprintf("exit code: %d, stdout: %s, stderr: %s", rr.ExitCode, rr.Stdout, rr.Stderr)
 }
 
+func RedactSensitiveData(msg string) string {
```

Comment:
Same here. 
Right now, this function is only used by `RunWithResult()`, so we don't need to export it.

We can consider creating a Redactor package in the future, which would take any `RedactData` and do the sanitization.  But that can be done in the future.